README.md

```diff
@@ -4,6 +4,7 @@ Reusable lightweight library component based on React.js
 ## Set up
 Installing all dependencies
 
+    $ npm install typescript@3.5.3 --save-dev
```

Comment:
doesn't this get done as part of the below package.json?

Reply:
I have done several tests and I have recently downloaded version 3.6.2 which generates the following error in the build:
https://github.com/microsoft/TypeScript/issues/32333
I found that forcing version 3.5.3 this error does not occur.

Reply:
Thanks for the recommendation, I already made the change